tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1648,6 +1648,58 @@ public void Edit_GameValid_GameEdited()
             VerifyEditGame(game, Times.Once());
         }
 
+        /// <summary>
+        /// Test for Edit method. Change time for game in second round with unknown teams. 
+        /// No exceprtion returns and game is edited successfully.
+        /// </summary>
+        [TestMethod]
+        public void Edit_ChangeTimeForPlayOffGameWithUnknownTeams_GameSaved()
+        {
+            //Arrange
+            var testTournament = CreatePlayoffTournament();
+            var games = new GameTestFixture()
+                .TestEmptyGamePlayoffSchedule()
+                .Build();
+
+            var testGameForEdit = GetTestGameForEditFromSecondRoundInPlayOff(games);
+
+            MockAllTournamentQueries(testTournament);
+            MockGetTournamentResults(TOURNAMENT_ID, games);
+
+            //Act
+            var sut = BuildSUT();
+            sut.Edit(testGameForEdit);
+
+            //Assert
+            VerifyEditGame(testGameForEdit, Times.Once());
+        }
+
+        /// <summary>
+        /// Test for Edit method. Change time for planned game in second round with defined teams. 
+        /// No exceprtion returns and game is edited successfully.
+        /// </summary>
+        [TestMethod]
+        public void Edit_ChangeTimeForPlayOffGameWithDefinedTeams_GameSaved()
```

Comment:
So what is the difference in those two tests? Both methods are identical

Reply:
Inattention. Repaired it.